### ISSUE TITLE
Website link rot

### ISSUE THREAD

**Issue by BotYue** [jem/jemdoc#23]
I recently noticed that the project website http://jemdoc.jaboc.net/ goes invalid. The webpage is now linking to some scam advertisement.

**Comment by rsha256** [comment]
yeah given the python2 requirement, I don't think this is being maintained anymore. Also see the last commit dates...
<img width="335" alt="image" src="https://github.com/jem/jemdoc/assets/28417128/6e0cdd66-772a-48de-bd10-651cc2f59b6e">
